protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2268,6 +2330,70 @@ func extractFieldBehaviorFromFieldDescriptor(fd *descriptorpb.FieldDescriptorPro
 	return opts, nil
 }
 
+func getFieldVisibilityOption(fd *descriptor.Field) *visibility.VisibilityRule {
+	if fd.Options == nil {
+		return nil
+	}
+	if !proto.HasExtension(fd.Options, visibility.E_FieldVisibility) {
+		return nil
+	}
+	ext := proto.GetExtension(fd.Options, visibility.E_FieldVisibility)
+	opts, ok := ext.(*visibility.VisibilityRule)
+	if !ok {
+		fmt.Fprintf(os.Stderr, "%s: extension is %T; want a *VisibilityRule object", fd.FQFN(), ext)
```

Comment:
I don't know if we even need to log this, this is a fundamental assumption error somewhere, no? Our tests should start failing if this fails, I think we can just ignore it?

Reply:
Great call. I have removed these.